docs/src/content/codesnippets/webhooks-security.txt

```diff
@@ -0,0 +1,12 @@
+import datetime
+import hmac
+
+request_timestamp = request.headers["X-Ultravox-Webhook-Timestamp"]
+if datetime.datetime.now() - datetime.dateimte.fromisoformat(request_timestamp) > datetime.timedelta(minutes=1):
```

Comment:
The last datetime here is misspelled as dateimte (which I assume was my fault typing this out in Slack :sweat_smile: )